symplyphysics/laws/quantum_mechanics/schrodinger/free_particle_plane_wave_solution.py

```diff
@@ -40,7 +46,42 @@
 
 law = Eq(wave_function, exp((I / hbar) * (particle_momentum * position - particle_energy * time)))
 
-# TODO: derive from Schrödinger equation
+# Derive from Schrödinger equation
+
+_time_independent_eqn = time_independent_eqn.law.subs(
+    time_independent_eqn.position, position,
+).subs({
+    time_independent_eqn.potential_energy(position): 0,
+    time_independent_eqn.particle_energy: kinetic_energy_via_momentum.law.rhs.subs({
+        kinetic_energy_via_momentum.mass: time_independent_eqn.particle_mass,
+        kinetic_energy_via_momentum.momentum: particle_momentum,
+    }),
+})
+
+_c1, _c2 = symbols("C_1 C_2")
+
+_time_independent_solution = dsolve(
+    _time_independent_eqn,
+    time_independent_eqn.wave_function(position),
+).rhs.subs({
+    "C1": _c1,
+    "C2": _c2,
+})
+
+_time_dependent_solution = time_dependent_law.law.rhs.subs({
+    time_dependent_law.time_independent_wave_function(time_dependent_law.position): _time_independent_solution,
+    time_dependent_law.particle_energy: particle_energy,
+    time_dependent_law.time: time,
+})
+
+# In general the solution of the Schrödinger equation can be viewed as the sum of waves traveling in opposite
+# directions.
+_current_law_solution = (
```

Comment:
Not justified. It seems, we've proven, that there are many solutions, for different `c1` and `c2`. But this law assumes some specific set of `c1` and `c2`.